src/main/kotlin/org/springframework/data/r2dbc/core/CriteriaStepExtensions.kt

```diff
@@ -15,7 +15,7 @@
  */
 package org.springframework.data.r2dbc.core
 
-import org.springframework.data.r2dbc.query.Criteria
+import org.springframework.data.relational.core.query.Criteria
```

Comment:
We should leave this extension as-is and rather add a Kotlin extension for `org.springframework.data.relational.core.query.Criteria` in Spring Data Relational.

Reply:
Thank you for your advice. I've dealt with it . PR #340